app.js

```diff
@@ -0,0 +1,256 @@
+const hamburger = document.querySelector('.hamburger');
+const menu = document.querySelector('#menu');
+const close_btn = document.querySelector('.close-btn');
+const porfolio_div = document.querySelector('#porfolio');
+const menu_items = document.querySelectorAll('.menu-item');
+const pop_container = document.querySelector('.pop-container');
+
+
+
+hamburger.addEventListener('click',()=>{
+menu.style.display='flex';
+});
+
+close_btn.addEventListener('click',()=>{
+    menu.style.display='none';
+});
+
+for(let menu_item of menu_items){
+    menu_item.addEventListener('click',()=>{
+        menu.style.display='none';
+    });
+}
+
+const getBtnId=(index)=>'btn-'+index;
+
+//list of projects in object
+const projects=[
+    {
+    name:"Facebook 360",
+    description:"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    featured_image:"images/snapshort_project1.png",
+    link_to_live:"https://abdallahmalima.github.io/portfolio_mobile_first/",
+    link_to_source:"https://github.com/abdallahmalima/portfolio_mobile_first",
+    technologies:["CANOPY","Back","2015"],
+    languages:["html","css","javascript","github","ruby","bootstrap"],
+   },
+   {
+    name:"Uber Navigation",
+    description:"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    featured_image:"images/snapshort_project2.png",
+    link_to_live:"https://abdallahmalima.github.io/portfolio_mobile_first/",
+    link_to_source:"https://github.com/abdallahmalima/portfolio_mobile_first",
+    technologies:["CANOPY","Back","2015"],
```

Comment:
- [ ] Great job. kindly ensure that the see live and see source buttons are well centralized in the mobile version